bigchaindb/web/views/base.py

```diff
@@ -23,7 +23,7 @@ def make_error(status_code, message=None):
 
 def base_ws_uri():
     """Base websocket uri."""
-    scheme = config['wsserver']['scheme']
-    host = config['wsserver']['host']
-    port = config['wsserver']['port']
+    scheme = config['wsserver']['advertised_scheme']
```

Comment:
I meant that it would be nice to put some more details in the comments at the top of this function definition, so users of the function know what it is for, i.e. when it is supposed to be used.